tests/test_examples_seq2seq.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
```

Comment:
nit: shouldn't this file be named `test_example_seq2seq`?

Reply:
I wouldn't mind if we consolidate with the example that dexter is making and just had `test_examples.py`

Reply:
Yeah, I like this - I'm not sure we need multiple test_example scripts at the moment

> I wouldn't mind if we consolidate with the example that dexter is making and just had `test_examples.py`

Yeah, I like this - I'm not sure we need multiple test_example scripts at the moment